data/json/vitamin.json

```diff
@@ -3,39 +3,64 @@
     "id": "calcium",
     "type": "vitamin",
     "name": "Calcium",
-    "min": -600,
-    "deficiency": [ [ "brittlebone", -300 ] ]
+    "deficiency": "hypocalcemia",
+    "min": -720,
+    "disease": [
+      [ -288, -336 ],
+      [ -337, -384 ],
+      [ -385, -720 ]
```

Comment:
> min and max are sufficient. There are values within that range that produce no diseases.

I didn't mean renaming. I meant thresholds for diseases, not possible ranges.

> Yes, it's reasonable to expect anyone implementing further vitamins to provide matching severity and diseases levels. I don't want to start performing micro-optimisations of the JSON until both the implementation is otherwise complete and there is evidence that we intend to add further vitamins.

It's a matter of non-redundancy, not a micro-optimisation. Your current storage format is extremely redundant (didn't you use calculations to split the range manually?). That provides a room for various errors and slows down balancing, but if you're sure, that your current format is more convenient, I'm not going to argue.